apps/accounts/forms.py

```diff
@@ -384,16 +391,84 @@ class ConsentForm(forms.ModelForm):
     data_processing_opt_in = forms.BooleanField(
         required=True,
         initial=False,
-        label="I consent to my submitted information being stored and processed to allow a response to my inquiry",
-        help_text="See our full privacy notice at https://www.thegreenwebfoundation.org/privacy-statement/",
+        label=(
+            "I consent to my submitted information being stored and processed to allow"
+            " a response to my inquiry"
+        ),
+        help_text=(
+            "See our full privacy notice at"
+            " https://www.thegreenwebfoundation.org/privacy-statement/"
+        ),
     )
     newsletter_opt_in = forms.BooleanField(
         required=False,
         initial=False,
         label="Sign me up to the newsletter",
-        help_text="We run a newsletter, Greening Digital, where we share actionable news about greening the web and a sustainable digital transition. You can unsubscribe at any time.",
+        help_text=(
+            "We run a newsletter, Greening Digital, where we share actionable news"
+            " about greening the web and a sustainable digital transition. You can"
+            " unsubscribe at any time."
+        ),
     )
 
     class Meta:
         model = ac_models.ProviderRequestConsent
         exclude = ["request"]
+
+
+class LocationForm(forms.ModelForm):
+
+    name = forms.CharField(
+        max_length=255,
+        label="Location name",
+        help_text=(
+            "Use the name you that your customers would recognise for this location."
+        ),
+        required=False,
+        widget=forms.widgets.TextInput(
+            attrs={
+                "placeholder": (
+                    "i.e. main headquarters for an office, or eu-west for a datacentre"
+                ),
+                "size": 60,
+            },
+        ),
+    )
+    description = forms.CharField(
+        label="Description",
+        help_text="If needed, add a description to help tell locations apart.",
+        required=False,
+        widget=forms.widgets.Textarea(attrs={"rows": 2, "cols": 80}),
+    )
+
+    city = forms.CharField(
+        max_length=255,
+        label="City",
+        help_text=(
+            "If this location is not within a given city, choose the nearest city."
+        ),
+    )
+
+    country = CountryField().formfield(
+        label="Country",
+        blank_label="Select country",
+        help_text="Choose a country from the list.",
+    )
+
+    class Meta:
+        model = ac_models.ProviderRequestLocation
+        # exclude = ["request"]
```

Comment:
It's safe to remove this commented-out line (`fields` and `exclude` are mutually exlusive)